### PR TITLE
maint: adjusting language on bulk enrollment steps descriptions

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
[Jira](https://openedx.atlassian.net/browse/ENT-4687)

screenshots:
**before**
step 1
Screen Shot 2021-06-24 at 4 17 31 PM

step 2
Screen Shot 2021-06-24 at 4 35 07 PM

step 3
Screen Shot 2021-06-24 at 4 42 38 PM

**after**

step 1
Screen Shot 2021-06-24 at 4 46 34 PM

step 2
Screen Shot 2021-06-24 at 5 05 57 PM

step 3
Screen Shot 2021-06-24 at 4 48 42 PM

### CODE REVIEW

File: src/components/BulkEnrollmentPage/stepper/ReviewStep.jsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ import { ADD_LEARNERS_STEP, ADD_COURSES_STEP, REVIEW_TITLE } from './constants';
 import ReviewList from './ReviewList';
 
 const EMAILS = {
```
Comment (raq929) [inline src/components/BulkEnrollmentPage/stepper/ReviewStep.jsx:9]: Heh, let's change the name of the constant as well.

File: src/components/BulkEnrollmentPage/stepper/AddCoursesStep.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const AddCoursesStep = ({
   goToNextStep, enterpriseId, enterpriseSlug, subscription,
 }) => (
   <>
-    <p>Find courses in your subscription catalog that matter most for your learners.</p>
+    <p>{ADD_COURSE_DESCRIPTION}</p>
```
Comment (raq929) [inline src/components/BulkEnrollmentPage/stepper/AddCoursesStep.jsx:21]: As you know, I love constants. <3

**Review by raq929 — APPROVED** [review]

File: src/components/BulkEnrollmentPage/stepper/ReviewStep.jsx (hunk 1 above)
Comment (alex-sheehan-edx) [reply src/components/BulkEnrollmentPage/stepper/ReviewStep.jsx:9]: hahaha good point

**Review by binodpant — APPROVED** [review]
would suggest tagging/FYI-ing Jen on this too but LGTM!

**Comment by joecassaro1** [comment]
New wording looks great! Excited that we are being so precise with our guidance to learners

**Pull request merged**